src/core/util/imagecache.cpp

```diff
@@ -45,7 +45,7 @@ void ImageCache::setMaster(std::shared_ptr<const Image> master) {
 }
 
 std::shared_ptr<const Image> ImageCache::getImage(const size2_t dimensions) const {
-    if (!master_) return std::shared_ptr<const Image>();
+    if (!master_ || (glm::compMul(dimensions) == 0)) return std::shared_ptr<const Image>();
```

Comment:
I think that this is more clear:
```suggestion
    if (!master_ || (glm::compMul(dimensions) == 0)) return nullptr;
```
Should be possible?

Reply:
In principle there is a syntactic difference between a nullptr (no data) and an empty Image (dimensions (0,0)).